src/intervalSequence.ts

```diff
@@ -1,122 +1,93 @@
-import * as sequence from './sequence'
+// import * as sequence from './sequence'
 import * as equal from './equal'
 import * as sign from './sign'
 
-// [edge, +Infinity)
-export type IntervalLeft<E, T> = {
-    readonly edge: E
+export type IntervalSequence<E, T> = {
```

Comment:
Do the two types `IntervalSequence` and `Edge` signify terminable and non terminable segments?